packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list.tsx

```diff
@@ -65,7 +66,7 @@ export const AccountPageList: FunctionComponent<AccountPageListProps> = ({
   currentPageEntityId,
   accountId,
 }) => {
-  const { data } = useAccountPages(accountId);
+  const { data, loading: pagesLoading } = useAccountPages(accountId);
```

Comment:
I realize this was what the hook returned already but from [this](https://github.com/hashintel/hash/pull/931#discussion_r958837057) comment in another PR we are trying to go with 
`[function, { loading, ...etc }]` instead. If you want I can make this change in that PR when this one gets merged.

Reply:
Using an array like that on hook return is more convenient for hooks that return an `action`, not the `data` afaik.

Reply:
Oh yeah you're right. And that's how Apollo does it which was the initial point anyway. Thanks for clearing that up